alica_engine/include/engine/containers/PlanTreeInfo.h

```diff
@@ -1,25 +1,29 @@
 #pragma once
 
-#include <vector>
 #include <tuple>
+#include <vector>
 
-using namespace std;
-
-namespace alica {
-typedef tuple<const supplementary::AgentID*, list<long>, list<long>> stdPlanTreeInfo;
-struct PlanTreeInfo {
+namespace alica
+{
+typedef std::tuple<const supplementary::AgentID*, std::vector<int64_t>, std::vector<int64_t>> stdPlanTreeInfo;
```

Comment:
IdGrp

Reply:
Actually, no. Ideally, this tuple definition stays "pure", i.e., it just happens by chance to have the same type as an IdGrp, but is not required to.